scripts/cleanup-dev-builds.sh

```
@@ -0,0 +1,249 @@
+    now="$(date +%s)"
+    echo $(( (now - mtime) / 86400 ))
+}
+
+# ---- safety probes ----------------------------------------------------------
+
+# Active tag (most recent reload) per the CLI symlink target.
+active_tag=""
+if [[ -r "$LAST_CLI_PATH_FILE" ]]; then
+    last_path="$(cat "$LAST_CLI_PATH_FILE" 2>/dev/null || true)"
+    if [[ "$last_path" == "$DERIVED_DATA_ROOT/cmux-"* ]]; then
+        rest="${last_path#$DERIVED_DATA_ROOT/cmux-}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)